crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -372,6 +372,7 @@ bitflags::bitflags! {
         const TRANSPARENT_MAIN_PASS       = (1 << 1);
         const MSAA_RESERVED_BITS          = MeshPipelineKey::MSAA_MASK_BITS << MeshPipelineKey::MSAA_SHIFT_BITS;
         const PRIMITIVE_TOPOLOGY_RESERVED_BITS = MeshPipelineKey::PRIMITIVE_TOPOLOGY_MASK_BITS << MeshPipelineKey::PRIMITIVE_TOPOLOGY_SHIFT_BITS;
+        const CULL_MODE_RESERVED_BITS = MeshPipelineKey::CULL_MODE_MASK_BITS << MeshPipelineKey::CULL_MODE_SHIFT_BITS;
```

Comment:
I think instead of shift bits, you could more simply do
```suggestion
        const CULL_MODE = (1 << 2);
```
I think MSAA has shift bits simply because it's expected to use the highest 6 bits. Would need to confirm with @superdump  to confirm this is correct.

Reply:
Interesting. So it looks like we currently have

```
VERTEX_TANGENTS
TRANSPARENT_MAIN_PASS
(unused)
(CULL_MODE_RESERVED_BITS)
PRIMITIVE_TOPOLOGY_RESERVED_BITS
MSAA_RESERVED_BITS
```

I had roughly followed what the primitive topology PR did. I'm not completely sure that I follow the rationale for placing the bits at one end or the other of the key though.

Reply:
The upper bits are being used for enums or other types packed into the bits of the key. So if you have just flags they can go into the lower bits. If you have an enum put it into the upper bits. And if using the upper bits, following the convention used for MSAA and such is good. Culling is only None/Back/Front isn’t it? *checks* yes. So it indeed needs two bits. The making and such all looks good. I have some other comments on the PR but they’re relatively minor. This PR is the kind of thing I had in mind. :) I’ll give it a proper review later.